includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -24,6 +24,22 @@ abstract class WC_Stripe_Payment_Gateway extends WC_Payment_Gateway_CC {
 	 */
 	protected $retry_interval = 1;
 
+	/**
+	 * Gateway label
+	 *
+	 * @var string
+	 */
+	protected $label;
```

Comment:
Since the label property will only be used to show admin notices, we can remove it and use the `method_title` property instead. It wouldn't make a difference for the store admin to see "Stripe Oxxo" instead of "Oxxo" in the message.

Reply:
I started off by using the `get_method_title()` method but I noticed it read significantly different than when UPE was enabled. So, I opted for adding the `get_label()` method, in order to maintain some naming consistency.

I just want to highlight that by using the `get_method_title()`, we get the following:

![image](https://user-images.githubusercontent.com/6844516/145836272-2f2f4a65-df6c-4bd8-a504-6bff23966bf7.png)

For comparison, when we enable UPE, we get the following:

![image](https://user-images.githubusercontent.com/6844516/145728537-ca209395-9137-4392-80ba-2f2ea74bdcfe.png)

I agree that it's easier to maintain the code if we use the `method_title` property so I will go ahead and make the change 👍